### PR TITLE
Updated urls for dropping center & institute registration

### PR DESCRIPTION
**Pull request by nishant22029**
-  Updated urls for dropping center and institute registration

### CODE REVIEW

File: wp-content/themes/goonj-crm/functions.php [hunk 1]
```diff
@@ -332,7 +339,7 @@ function goonj_custom_rewrite_rules() {
 	);
 
 	add_rewrite_rule(
-		'^actions/dropping-center/([0-9]+)/?',
+		'^actions/dropping-center/location/([0-9]+)/?',
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/functions.php:342]: I don't think this should be changed

File: wp-content/themes/goonj-crm/functions.php [hunk 2]
```diff
@@ -269,7 +276,7 @@ function goonj_handle_user_identification_form() {
 		->execute();
 
 		if ($purpose === 'dropping-center') {
-			wp_redirect(get_home_url() . "/dropping-center/#?Collection_Camp_Core_Details.Contact_Id=" . $found_contacts['id']);
+			wp_redirect(get_home_url() . "/dropping-center/location/#?Collection_Camp_Core_Details.Contact_Id=" . $found_contacts['id']);
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/functions.php:279]: Why have you named it location?

File: wp-content/themes/goonj-crm/functions.php [hunk 3]
```diff
@@ -182,6 +182,13 @@ function goonj_handle_user_identification_form() {
 			'not-inducted-for-dropping-center'
 		);
 
+		$institute_registration_individual_signup_form_path = sprintf(
```
Comment (pokhiii) [inline wp-content/themes/goonj-crm/functions.php:185]: can we use a single name either `registration` or `sign up`. I think we should use just registration

Comment (pokhiii) [reply wp-content/themes/goonj-crm/functions.php:185]: also I don't think we should create individual-sign-up as a child page of institute registration. they are independent

**Review by pokhiii — CHANGES_REQUESTED** [review]

File: wp-content/themes/goonj-crm/functions.php (hunk 2 above)
Comment (nishant22029) [reply wp-content/themes/goonj-crm/functions.php:279]: In dropping center landing page after email & phone details submitted existing user is redirected to fill location details for dropping center

File: wp-content/themes/goonj-crm/functions.php (hunk 3 above)
Comment (nishant22029) [reply wp-content/themes/goonj-crm/functions.php:185]: sure will update

**Pull request closed without merging**